CatProducer/prod/PAT2CAT_cfg.py

```diff
@@ -36,6 +36,9 @@
 from CATTools.CatProducer.catTools_cff import *
 catTool(process, runOnMC, doSecVertex, useMiniAOD)
 
+from CATTools.CatProducer.catGetHFHadronMatching_cff import *
+genHFTool(process, runOnMC,  useMiniAOD)
```

Comment:
a bit pedantic, but this shouldnt need runOnMC as input since it should be only used for mc